README.md

```diff
@@ -44,26 +44,50 @@ How evolved is the concept (see [Wardley Maps](https://medium.com/wardleymaps)):
 - genesis: new unexplored domain
 - custom built: companies are building their own versions
 - product: off-the-shelf versions exist with differentiation
-- commodity: highly-standardised versions exist 
+- commodity: highly-standardised versions exist
```

Comment:
Maybe we can add a link to the table with the stages of activities to help people reason about it: https://twitter.com/swardley/status/989211014485901316

Reply:
Great idea.